GW2EIEvtcParser/EIData/Buffs/BuffImages.cs

```diff
@@ -137,7 +137,7 @@ internal static class BuffImages
         internal const string SuperiorSigilOfBounty = "https://wiki.guildwars2.com/images/f/f8/Superior_Sigil_of_Bounty.png";
         internal const string MinorSigilOfBenevolence = "https://wiki.guildwars2.com/images/f/ff/Minor_Sigil_of_Benevolence.png";
         internal const string MajorSigilOfBenevolence = "https://wiki.guildwars2.com/images/c/c8/Major_Sigil_of_Benevolence.png";
-        internal const string SuperiorSigilOfBenevolence = "https://wiki.guildwars2.com/images/9/91/Superior_Sigil_of_Sanctuary.png";
+        internal const string SuperiorSigilOfSanctuary = "https://wiki.guildwars2.com/images/9/91/Superior_Sigil_of_Sanctuary.png";
```

Comment:
Why the renaming? Sanctuary sigil has not been in the game since 2014

Reply:
i've been trying to keep the const names the same as the image names, so it's easier to recognise if an image has been used before or not (other than searching for it obviously)

Reply:
I understand but sigil of sanctuary does not exist anymore, the naming disparity is more on the wiki than EI, even more so when they have properly named the minor and major correctly. I'd rather the names be what they represent in EI rather than be based on a third party's naming decisions.

Reply:
i've restored to the previous name